compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -88,13 +88,13 @@ object Types {
 
 // ----- Tests -----------------------------------------------------
 
-    // debug only: a unique identifier for a type
-    val uniqId = {
-      nextId = nextId + 1
+//    // debug only: a unique identifier for a type
+//    val uniqId = {
```

Comment:
Why did this get turned off?

Reply:
It's a field in every type, and we don't need it - it's used nowhere.